api/v1/routes/payment.py

```diff
@@ -0,0 +1,14 @@
+from fastapi import APIRouter, HTTPException, Depends
+from sqlalchemy.orm import Session
+from api.v1.services.payment import PaymentService
+from api.utils.dependencies import get_db
+from api.v1.schemas.payment import PaymentResponse
+from api.utils.db_validators import check_model_existence
+
+payment = APIRouter()
+
+@payment.get("/payments/{payment_id}", response_model=PaymentResponse)
+async def get_payment(payment_id: str, db: Session = Depends(get_db)):
+    payment_service = PaymentService()
+    payment = check_model_existence(db, payment_service.model, payment_id)
```

Comment:
This should be in services and should be like this
`check_model_existence(db, Payment, payment_id)` where Payment is the payment database model

Reply:
I have refactored to include check_model_existence in services and call it in get_payment_by_id inside routes